File: 242-ValidAnagram/242-ValidAnagram.js
// Last updated: 24/04/2026, 10:05:27
/**
 * @param {string} s
 * @param {string} t
 * @return {boolean}
 */
var isAnagram = function(s, t) {
  let sorted = s.split("").sort().join("");
    let sorted1 = t.split("").sort().join("");

    if(sorted == sorted1){
        return true;
    }
    else{
        return false;
    }
};

File: 242-ValidAnagram/242-ValidAnagram.cpp
// Last updated: 07/06/2026, 14:36:29
1class Solution {
2public:
3    bool isAnagram(string s, string t) {
4      sort(t.begin(), t.end());
5      sort(s.begin(), s.end());
6
7      return s == t;
8    }
9};